openehr-rm/src/main/java/com/nedap/archie/rm/datastructures/ItemList.java

```diff
@@ -34,4 +54,27 @@ public void addItem(Element item) {
         this.items.add(item);
         setThisAsParent(item, "items");
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        if (!super.equals(o)) return false;
+
+        ItemList itemList = (ItemList) o;
+
+        if (items != null && itemList.items == null) return false;
+        if (items == null && itemList.items != null) return false;
+
+        // Compering elements ignoring order
+        return items.equals(itemList.items);
```

Comment:
If items == null and itemList.items == null, this will throw a NullPointerException instead of returning true

Also List.equals does a compare that does __not__ ignore order. I think that is correct and it should preserve order, as ITEM_LIST is an ordered list, usually.

very minor: comparing instead of compering